### PR TITLE
Range input colors

### PR DESCRIPTION
**Pull request by cfarm**
Adds branded colors for range input, see https://github.com/cfpb/cf-forms/pull/43 for details

### CODE REVIEW

File: app/templates/src/static/css/cf-theme-overrides.less [hunk 1]
```diff
@@ -134,9 +134,17 @@
 // .success
 @input-success:                 @green;
 
+// .disabled
+@input-disabled:                 @gray-10;
```
Comment (jimmynotjim) [inline app/templates/src/static/css/cf-theme-overrides.less]: Should probably keep this one or [this one](https://github.com/cfpb/generator-cf/pull/102/files#diff-135a296121653a13551c1aeb70adc9b0R138), although they may just cancel each other out once one of these is merged.

**Comment by jimmynotjim** [comment]
:+1:

**Comment by Scotchester** [comment]
Merge conflict needs fixing, please!

**Comment by Scotchester** [comment]
Also, on hold pending the resolution of [this comment on a related PR](https://github.com/cfpb/cf-forms/pull/43/files#r41658498).

**Comment by anselmbradford** [comment]
Is this still applicable?

**Comment by jimmynotjim** [comment]
We never ended up implementing range inputs, we might want to do that first.

**Pull request closed without merging**